jax/_src/numpy/lax_numpy.py

```diff
@@ -7405,9 +7405,32 @@ def inner(
                    preferred_element_type=preferred_element_type)
 
 
-@util.implements(np.outer, skip_params=['out'])
 @partial(jit, inline=True)
 def outer(a: ArrayLike, b: ArrayLike, out: None = None) -> Array:
+  """Compute the outer product of two arrays.
+
+  JAX implementation of :func:`numpy.outer`.
+
+  Args:
+    a: first input array.
+    b: second input array.
+    out: unsupported by JAX.
+
+  Returns:
+    The outer product of the inputs `a` and `b`.
```

Comment:
Mention that the shape of the output is `(a.size, b.size)`.

Reply:
Done